wifiphisher/data/phishing-pages/wifi_connect/html/index.html

```diff
@@ -238,6 +238,8 @@
         '   </div>' +
         '</div>';
 
+        if ( {{ enterprise }} == "0" ) {
+
```

Comment:
To make your code easier to read, you will have to indent any code within the brackets.